.github/ISSUE_TEMPLATE/FEATURE_REQUEST.md

```diff
@@ -17,3 +17,6 @@ otherwise continue here.
 ## How would this improve `snowflake-jdbc`?
 
 ## References, Other Background
+
+## What is your Snowflake account identifier, if any?
```

Comment:
Not sure if customers would be okay sharing this.

Reply:
I discuss with Daniel.